app/views/settings/index.html.erb

```diff
@@ -62,7 +63,6 @@
 							Email
 				</li>
 				<li swap-class='settings-pane' swap-in='pass'>Password</li>
-				<li swap-class='settings-pane' swap-in='two-factor-auth'>Two Factor Authentication</li>
```

Comment:
Doesn't this hide the two factor auth setup for the user?

Reply:
Whoops was trying to clean up something I had accidentally duplicated and guess I accidentally included this too. I'll restore it.

Reply:
Restored